pkg/kubelet/volume/gce_pd/gce_pd.go

```diff
@@ -151,6 +151,13 @@ type gcePersistentDisk struct {
 	legacyMode bool
 }
 
+func detachDisk(pd *gcePersistentDisk) {
```

Comment:
Is it worth naming this something like detachDiskLogError() to differentiate it from the path in TearDown() which does not log the error but returns it.